workflows/rnaseq/main.nf

```diff
@@ -866,11 +870,31 @@ workflow RNASEQ {
         ch_multiqc_files = ch_multiqc_files.mix(ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'))
         ch_multiqc_files = ch_multiqc_files.mix(ch_collated_versions)
 
+        // Derive a set of rename patterns such that any sample names derived
+        // from input fastqs in MultiQC are replaced with specified sample name
+        // strings
+
+        ch_name_replacements = ch_samples
+            .map{
+                def read1 = it[1].name.split('\\.')[0] + "\t" + it[0].id + '_1'
+                def read2 = ''
+                if (it[2] ){
+                    read2 = it[2].name.split('\\.')[0] + "\t" + it[0].id + '_2'
+                }
+                return [[read1, read2]]
+            }
+            .transpose()
```

Comment:
Since I have no clue about Groovy, I don't feel competent to review this. 
In particular, thinking through how the `.transpose()` result would look like after swapping the rows with columns is beyond my comprehension for cases with empty strings and missing items etc.

Reply:
FWIW I was using transpose as a way of converting:

```
[[name1], [name2]]
```

... to

```
[[name1]]
[[name2]]
```

(but I've simplified to use flatten() instead).